validator/keymanager/remote-web3signer/internal/client_test.go

```diff
@@ -125,6 +125,23 @@ func TestClient_GetPublicKeys_HappyPath(t *testing.T) {
 	assert.EqualValues(t, "[162 181 170 173 156 110 254 254 123 185 177 36 58 4 52 4 243 54 41 55 207 182 179 24 51 146 152 51 23 63 71 102 48 234 44 254 176 217 221 241 95 151 202 134 133 148 136 32]", fmt.Sprintf("%v", resp[0][:]))
 }
 
+func TestClient_GetPublicKeys_EndoingError(t *testing.T) {
```

Comment:
Is endoing a word?

Reply:
it was supposed to be encoding 😮‍💨  no idea how it changed to that